pages/Category.vue

```diff
@@ -545,9 +538,9 @@ export default {
       return aggregations
         .reduce((result, aggregation) => {
           const bucket =
-            this.aggregations &&
-            this.aggregations[aggregation] &&
-            this.aggregations[aggregation].buckets.find(
+            this.getAggregations &&
+            this.getAggregations[aggregation] &&
+            this.getAggregations[aggregation].buckets.find(
```

Comment:
Please add `Array.isArray(this.getAggregations[aggregation].buckets)` check too

Reply:
Added the check for the aggregations array